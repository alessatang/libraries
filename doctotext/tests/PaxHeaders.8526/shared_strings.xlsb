30 mtime=1377256131.814675643
30 atime=1389151483.452844604
30 ctime=1389151483.452844604
