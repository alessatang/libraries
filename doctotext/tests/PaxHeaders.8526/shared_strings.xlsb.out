29 mtime=1379553523.78759009
30 atime=1389151483.468844537
30 ctime=1389151483.469844533
